src/solver.jl

```diff
@@ -15,15 +16,15 @@ immutable PajaritoSolver <: MathProgBase.AbstractMathProgSolver
     opt_tolerance               # Relatice optimality tolerance
     time_limit                  # Time limit
     profile::Bool               # Performance profile switch
-    disaggregate_soc::Symbol    # SOCP disaggregator for SOC constraints
+    disaggregate_soc            # SOCP disaggregator for SOC constraints
     instance::AbstractString    # Path to instance
     enable_sdp::Bool            # Indicator for enabling sdp support
     force_primal_cuts::Bool     # Enforces primal cutting planes under conic solver
     dual_cut_zero_tol::Float64  # Tolerance to check if a conic_dual is zero or not
 end
 
-function PajaritoSolver(;verbose=0,algorithm="OA",mip_solver=MathProgBase.defaultMIPsolver,cont_solver=MathProgBase.defaultNLPsolver,opt_tolerance=1e-5,time_limit=60*60*10,profile=false,disaggregate_soc=:default,instance="",enable_sdp=false,force_primal_cuts=false,dual_cut_zero_tol=0.0)
-    disaggregate_soc_ind = Symbol(string(disaggregate_soc))
+function PajaritoSolver(;verbose=0,algorithm="OA",mip_solver=MathProgBase.defaultMIPsolver,cont_solver=MathProgBase.defaultNLPsolver,opt_tolerance=1e-5,time_limit=60*60*10,profile=false,disaggregate_soc=_disagg_soc_default,instance="",enable_sdp=false,force_primal_cuts=false,dual_cut_zero_tol=0.0)
+    disaggregate_soc_ind = (typeof(disaggregate_soc) == Bool && disaggregate_soc) ? _disagg_soc_on : _disagg_soc_off
```

Comment:
you can write ``disaggregate_soc === true`` instead of ``typeof(disaggregate_soc) == Bool && disaggregate_soc``